src/index.css

```diff
@@ -0,0 +1,57 @@
+body {
+  width: 100vw;
+  height: 100vh;
+  margin: 0;
+  display: flex;
+}
+
+* {
+  margin: 0;
+}
+
+#root {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] To comply with DRY and CSS best practices, kindly consider not repeating the same rule more than once. for example instead of taking `display: flex` multiple times in multiple selectors, you can add a class like this:
```
.flex {
display: flex;
}
```
and then give `flex` class to any element you want. you can repeat this pattern for all the other similar instances.